backend/src/services/services.ts

```diff
@@ -71,6 +73,11 @@ const space = new Space({
   s3Service: s3,
 });
 
+const slc = new Function({
```

Comment:
```suggestion
const function = new Function({
```
i think should be the same

Reply:
'function' is not allowed as a variable declaration name.ts(1389) :/